README.md

```diff
@@ -39,13 +39,47 @@ Building and running with Docker
 `make run`
 
 
-Launch migration
------------------------------------
+Launch images migration from V5 to V6
+-------------------------------------
+
+Migration take source images from v5 S3 read-only bucket, which could be
+defined by environment variables, example:
+
+```
+V5_BUCKET=c2corg_images_master
+V5_ENDPOINT=https://sos.exo.io
+V5_PREFIX: EXO
+EXO_ACCESS_KEY_ID: ...
+EXO_SECRET_KEY: ...
+```
+
+Note that here, ``PREFIX`` point out the keys to use as we can have multiple
+endpoints (AWS, Exoscale) with different keys.
+
+The migration script iterate through v5 images. For each *original* image found:
+* If the image already exists on publication bucket, nothing is done (only
+  migrate the new ones).
+* If the image do not exists on the v6 bucket:
```

Comment:
We 4 storages : v5_storage, temp_storage, incoming_storage and active_storage.

In the migration script, active_storage is called v6 bucket for the distinction with v5 one.

In the resize_images script is is called publication bucket for the distinction with incoming bucket.